docs/i18n.md

```diff
@@ -0,0 +1,93 @@
+# Internationalization (i18n)
+Stratos UI supports i18n via Angular's [$translate service](https://github.com/angular-translate/angular-translate). Dates and times are handled by [Moment](https://momentjs.com/)'s i18n process.
+
+The locale shown will at first be selected from the browser. Any change of locale by the user will be retained within the browser's session. 
+
+The source for all translations can be found in the relevant plugin directories i18n folder. For examples
```

Comment:
The source for all translations can be found in the relevant component's i18n folder. For example: